### DISCUSSION TITLE
grapesjs version v0.18 to v0.21

### DISCUSSION THREAD

**Discussion by aharishsundhar** [GrapesJS/grapesjs#5124] [General]
@artf if your free please ask small question 
previously from the blocksmanager used content as per use html, style, script tag it has to be drag and drop well fine to use preview editing
For now I am using v0.21 grapesjs it will update happen not working yet on style and script tag
How can I put in style and script in custom blocks?

Thanks and regards
Harishsundhar

**Comment by artf** [comment]
You have to use custom [components](https://grapesjs.com/docs/modules/Components.html), using styles and scripts in blocks it's not correct
